ang/civicase/myactivities/directives/crm-dashboard-tabset-affix.directive.js

```diff
@@ -0,0 +1,61 @@
+(function (angular, $) {
```

Comment:
I think this should be inside a `ang/civicase/crm-dashboard` folder.

Reply:
But tbh, this should be a generic implementation if possible. Not sure how long it would take, but I see that we repeat the same pattern.

I think we can have something like:

```
<div civicase-affix-after="civicase__crm-dashboard">...</div>
```

I'm not sure about all these calculations, but isn't it enough to know only how far from the top the main container element is?

Reply:
I agree, and I know I have been ignoring this particular bit, but honestly this would take much longer time if I start refactoring all directives now. then we wont be able to finish it by tomorrow, as the testing scope will increase a lot too.